### DISCUSSION TITLE
How do OBS handle reconnection to a device?

### DISCUSSION THREAD

**Discussion by emre570** [obsproject/obs-studio#11572] [General]
Hello I'm a new beginner. I'm trying to build a camera controller for handling multiple static ip assigned cameras. I'm struggling to implement reconnection logic. I tried using OBS and went curious about how did you do. Is there any hands-on concepts or techniques about it?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
